.github/workflows/deployment.yml

```diff
@@ -109,7 +109,7 @@ jobs:
       - uses: actions/checkout@v1
         with:
           repository: AltaPay/plugin-wordpress
-          ref: 'origin/main'
+          ref: '${{env.TEST_BRANCH_NAME}}'
```

Comment:
Is this correct?

Reply:
Yes it is. This variable has the current branch name being merged to run the integration tests from.